test/unit/plugins/transform-dot-component-invocation.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const Linter = require('../../../lib/index');
+const expect = require('chai').expect;
+
+describe('transform dot component invocation', function() {
+  function buildFakeConsole() {
+    return {
+      _logLines: [],
+
+      log(data) {
+        this._logLines.push(data);
+      }
+    };
+  }
+
+  let mockConsole;
+  beforeEach(function() {
+    mockConsole = buildFakeConsole();
+  });
+
+  function ensureValid(source) {
+    it(`passes with: \`${source}\``, function() {
+      let linter = new Linter({
+        console: mockConsole
+      });
+
+      expect(linter.verify({ source, moduleId: 'some/thing.hbs' })).to.deep.equal([]);
+    });
+  }
+
+  ensureValid('{{#foo-bar as |baz|}}{{#baz.derp}}{{/baz.derp}}{{baz.derp}}{{/foo-bar}}');
```

Comment:
Ahh, sorry about that! I was actually meaning that you would add just this line to the `test/unit/recommended-config-test.js` file.